centreon/tests/e2e/cypress/e2e/OpenID-connect/01-oidc-authentication/index.ts

```diff
@@ -49,7 +49,7 @@ When(
     cy.getByLabel({ label: 'Identity provider' })
       .eq(0)
       .contains('Identity provider')
-      .click();
+      .click({ force: true });
```

Comment:
Why forcing the click?

Reply:
@Thebarda sometimes it doesn't find the element even tho it exists in the dom so this fixes it.